### ISSUE TITLE
Parent config generator labels to their respective inputs

### ISSUE THREAD

**Issue by PitchforkAssistant** [PitchforkAssistant/devvit-flair-assistant#1]
The HTML structure for the config generator is currently: 

```
<div>
    <label></label>
    <input>
</div>
```

Ideally it should be:

```
<label>
    <input>
</label>
```

**Comment by PitchforkAssistant** [comment]
Fixed by 37575b8f48b8a67cf8b27a7d110e992436d27bc2

**Comment by PitchforkAssistant** [comment]
Fixed with 37575b8f48b8a67cf8b27a7d110e992436d27bc2
